content/chronograf/v1.8/tools/chronograf-cli/_index.md

```diff
@@ -84,15 +84,15 @@ chronograf [flags]
 |:-------------------------------|:------------------------------------------------------------------------|:--------------------|
 | `-i`, `--github-client-id`     | GitHub client ID value for OAuth 2.0 support                            | `$GH_CLIENT_ID`     |
 | `-s`, `--github-client-secret` | GitHub client secret value for OAuth 2.0 support                        | `$GH_CLIENT_SECRET` |
-| `-o`, `--github-organization`  | Specify a GitHub organization membership required for a user. Optional. | `$GH_ORGS`          |
+| `-o`, `--github-organization`  | Restricts authorization to users from comma-separated list of specified Github organizations. Optional. | `$GH_ORGS`          |
 
 ### Google-specific OAuth 2.0 authentication flags
 
 | Flag                     | Description                                                                     | Env. Variable           |
 |:-------------------------|:--------------------------------------------------------------------------------|:------------------------|
 | `--google-client-id`     | Google client ID value for OAuth 2.0 support                                    | `$GOOGLE_CLIENT_ID`     |
 | `--google-client-secret` | Google client secret value for OAuth 2.0 support                                | `$GOOGLE_CLIENT_SECRET` |
-| `--google-domains`       | Restricts authorization to users from specified Google email domains. Optional. | `$GOOGLE_DOMAINS`       |
+| `--google-domains`       | Restricts authorization to users from comma-separated list of specified Google email domains. Optional. | `$GOOGLE_DOMAINS`       |
```

Comment:
I don't think this matches what the underlying issue outlines.  These are flags... and when using flags it implies having multiple flags.  

When using environment variables --> then the comma-separated list works.